### ISSUE TITLE
Autocompletion for replication commands is lacking

### ISSUE THREAD

**Issue by datacharmer** [dbcli/mycli#124]
I see that almost everything related to replication is missing in the autocompletion:

`CHANGE MASTER TO` ([MariaDB 10.x](https://mariadb.com/kb/en/mariadb/change-master-to/) )
`SHOW ALL SLAVES STATUS` ([MariaDB 10.x](https://mariadb.com/kb/en/mariadb/multi-source-replication/))
`SHOW SLAVE connection_name STATUS` (MariaDB 10.x)
`START/STOP [connection_name] SLAVE`

`CHANGE MASTER TO` ([MySQL 5.7](https://dev.mysql.com/doc/refman/5.7/en/change-master-to.html)) 
`SHOW SLAVE STATUS [FOR CHANNEL connection_name]` ([MySQL 5.7](https://dev.mysql.com/doc/refman/5.7/en/replication-slave-sql.html))
`START/STOP SLAVE [for channel ...]`
`CHANGE REPLICATION FILTER`

**Comment by amjith** [comment]
You're right it is lacking. That is one area I need to improve. I'll add a new completion category and see if I can make this work. Thanks for the links to docs. :)

**Comment by amjith** [comment]
@datacharmer I've implemented the completions for replication related statements. Please let me know if anything is still missing. 

The changes are available in master and you can try it out by installing directly from master as follows: 

```
pip install -U https://github.com/dbcli/mycli/archive/master.zip
```

**Comment by datacharmer** [comment]
I see that the at least the following statements are not in the autocompletion:

    # mysql 5.7
    CHANGE MASTER TO  MASTER_AUTO_POSITION=1 for channel 'xxx'
    START/STOP SLAVE FOR CHANNEL 'xxx'

    # mariadb 10.x  
    CHANGE MASTER 'yyy' TO MASTER_USE_GTID=current_pos
    START/STOP slave 'yyy'
    SHOW ALL SLAVES STATUS

**Comment by amjith** [comment]
Looks like I shouldn't blindly copy the terms in for the suggestion candidates. I hadn't anticipated the difference between MariaDB and MySQL. I might run a help command and try to parse the output to populate the possible candidates. Thank you for testing it and providing quick feedback. I'll work on it more and get back to you.

**Comment by rolandwalker** [comment]
It looks like @amjith made some improvements in this issue back in 2015, and we just improved completions further in #1447.    I'm not sure we should prioritize going deeper into the different MySQL versions, when there are other places in the completions which need more work (for statements entered more frequently than replication commands).

Therefore I'm closing, though anyone should feel free to re-open.
